examples/fluent-bit/kinesis-stream/task-definition.json

```diff
@@ -0,0 +1,39 @@
+{
+	"family": "firelens-example-data-stream",
```

Comment:
nit: maybe we can include the keyword `kinesis` in the name, like `firelens-example-kinesis-data-stream`.

Reply:
Done